articles/azure/azs-how-create-vm.md

```diff
@@ -1,20 +1,19 @@
 ---
-title: How to create a virtual machine from the Azure Marketplace| UKCloud Ltd
+title: How to create a Virtual Machine - Portal| UKCloud Ltd
```

Comment:
How to create a virtual machine using the Azure Stack Portal

Reply:
Fixed